### DISCUSSION TITLE
How do I apply for student certification if I don't have an education email

### DISCUSSION THREAD

**Discussion by muhuizz** [community/community#110090] [GitHub Education]
### Hi everyone,

The school only offers the option to apply for an educational email for graduate students. I am currently pursuing my undergraduate degree. Are there any other ways to apply for educational certification?

**Comment by SebastienXia** [comment]
![GITHUB EDUCATION BANNER](https://user-images.githubusercontent.com/97467892/151880377-1c26ae86-8fbf-4874-9bb3-af7c7d050486.png)

## Hi, @muhuizz and welcome to Github Community 😉

There is an other option to apply, with a **personal email**, like I did with mine ! user@gmail.com (Just make sure you meet the conditions and documents requested during the process)


`close the thread and mark as answered if this helped you resolve your issue.`
